.doc_gen/metadata/rds_metadata.yaml

```diff
@@ -0,0 +1,104 @@
+
+#zexi 0.4.0
+rds_CreateDBInstance:
+  title: Create an &RDSlong; instance and wait for it to be in an available state using an &AWS; SDK
+  title_abbrev: Create an instance
+  synopsis: create an &RDS; instance.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/rds
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - rds.java2.create_instance.main
+  services:
+    rds: {CreateDBInstance}
+rds_DeleteDBInstance:
+  title: Delete an &RDSlong; instance using an &AWS; SDK
+  title_abbrev: Delete an instance
+  synopsis: delete an &RDS; instance.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/rds
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - rds.java2.delete_instance.main
+  services:
+    rds: {DeleteDBInstance} 
+rds_DescribeAccountAttributes:
+  title: Retrieve attributes that belongs to an &RDSlong; account using an &AWS; SDK

```

Comment:
&RDSlong; -> &RDS;
attributes that **belong** to...

Reply:
fixed